src/common/ossl_helpers.c

```diff
@@ -114,7 +118,7 @@ const EVP_MD *oqs_sha256(void) {
 #endif
 	return sha256_ptr;
 #else
-	return OSSL_FUNC(EVP_sha256)();
+	return EVP_sha256();
```

Comment:
Why is the macro being removed here? And similarly on the rest of the diff in this file?